theme/helpers/_zindex.scss

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+// For licensing, see LICENSE.md or http://ckeditor.com/license
+
+$ck-def-z: (
+	'default': 1
+	// 'modal': 10
```

Comment:
This helper will be developed as more and more components will join the default library. Comments show the general course of development and idea behind it. It will look like that in the nearest future.